docs/users.rst

```diff
@@ -1,15 +0,0 @@
-=====
```

Comment:
Let's keep it and add a few entries about FxA here and links to the KB articles.